app/Pivots/EventParticipant.php

```diff
@@ -27,4 +28,18 @@ public function package()
     {
         return $this->belongsTo(EventPackage::class);
     }
+
+    public function isPaid(): bool
+    {
+        return $this->datum_betaling !== '0000-00-00';
+    }
+
+    public function getPayment(bool $existing = true): EventPayment
```

Comment:
See previous comment about this.

```suggestion
    public function makePayment(): EventPayment
```